realm/realm-library/src/main/java/io/realm/OrderedRealmCollection.java

```diff
@@ -137,4 +137,14 @@
      * @throws UnsupportedOperationException if the collection is unmanaged.
      */
     boolean deleteLastFromRealm();
+
+    /**
+     * Creates a snapshot from this {@link OrderedRealmCollection}.
+     *
+     * @return the snapshot of this collection.
+     * @see OrderedRealmCollectionSnapshot
+     * @throws java.lang.IllegalStateException if the Realm is closed or the method is called from the wrong thread.
+     * @throws UnsupportedOperationException if the collection is unmanaged.
+     */
+    OrderedRealmCollectionSnapshot<E> createSnapshot();
```

Comment:
Still debating with my myself whether `createSnapshot`, `getSnapshot` or `asSnapshot` is better. `createSnapshot` is probably fine though and also indicate that it is not a "free" method to call.